src/cache-manager.js

```diff
@@ -83,9 +106,9 @@ class CacheManager {
             this.debug(`error handling put request ${err.stack || err}`);
             done(err);
           });
-
         } else {
           this.debug('received empty sqs message, ignoring');
+          return done(new Error('no body present on sqs message'));
```

Comment:
Unrelated to this PR (don't let it block you), but please do consider early returns...
```js
if (!message.Body) {
  this.debug('received empty sqs message, ignoring');
  return done(new Error('no body present on sqs message'));
}

...
```

That would have saved a lot of indentation that isn't necessary.